woocommerce/woocommerce-ios
# [Custom Amounts M3] Order Details

Closes: #10885 


## Description


With this PR we update the order details screen to include custom amounts and implement other tweaks in consonance with the changes performed in the editable order screen.

## Changes

_See screenshots below fo a better appraisal_

- The products section title is moved to native instead of being embedded in the section itself unless we can add shipping labels. (we need the three dots button there).
- The custom amounts section has been added.
- The section header titles are removed for notes and shipping details when empty. "+ Add item ..." rows have larger height.
- The Add Note and Add Tracking rows have larger heights and the plus button icon aligns with the rest of the screen.

## Testing instructions


1. Go to orders
2. Tap on a created order. See below for each case:

### New product header and custom amounts section:

#### Before
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/8edc07f9-917e-42a2-8c0b-2b24b65cfc72" width="375">

### After
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/7e5f1984-a7c9-4c22-89ad-71bd952481d3" width="375">

#### Both: Shipping Labels button
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/78eb0b41-711e-4c5d-bd67-0ed3fbb1d4a3" width="375">

### No section headers when empty • Thicker rows:

#### Before:
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/17d9227d-67e8-4efb-8c26-02b81e1b19d2" width="375">

#### After:
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/be01ad9e-60b1-424e-967b-1669940c01d7" width="375">

## Screenshots



---
